daras_ai_v2/gooey_builder.py

```
@@ -225,3 +220,118 @@ def get_conversation_data_for_saved_run(
+            builder_page_cls.RequestModel, builder_sr.state
+        )
+    builder_sr = builder_pr.submit_api_call(
+        current_user=request.user,
+        workspace=workspace,
+        request_body=request_body,
+    )[1]
+
+    # Snapshot the SavedRun at workflow_url into a new run linked to
+    # builder_sr so the chat widget can navigate the user to a workflow page
+    # that knows which builder iteration produced it
+    workflow_page_cls, workflow_sr, workflow_pr = url_to_runs(body.workflow_url)
```

Ruff (0.15.14)

[warning] Unpacked variable `workflow_page_cls` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)